src/Model/EditCounter.php

```diff
@@ -469,7 +470,9 @@ public function countPagesProtected(): int
     public function countPagesReprotected(): int
     {
         $logCounts = $this->getLogCounts();
-        return $logCounts['protect-modify'] ?? 0;
+        return ($logCounts['protect-modify'] ?? 0)
+            + ($logCounts['stable-modify'] ?? 0)
+            + ($logCounts['stable-move_stable'] ?? 0);
```

Comment:
`move_stable` I think is just logged when the PC protection is moved to another title during a page move. Thus, we shouldn't count those. We don't count `protect-move_prot` for the same reason.